redaxo/src/core/schemas/config.json

```diff
@@ -300,7 +300,7 @@
             "type": "string"
         },
         "socket_proxy": {
-            "description": "Proxy used for all rex_socket connections",
+            "description": "Proxy used for all \\Redaxo\\Core\\HttpClient\\Request connections",
```

Comment:
Überlege, ob wir den Config-Key auch gleich umbenennen. `http_client_proxy` oder so?

In der Beschreibung ist die Frage, ob wir da den vollen Klassennamen wirklich brauchen. Vielleicht auch sowas wie "for all HttpClient connections"?